Naming/SubdirDirectoryPropertyNamer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Vich\UploaderBundle\Naming;
+
+use Vich\UploaderBundle\Mapping\PropertyMapping;
+use Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException;
+use Vich\UploaderBundle\Exception\NameGenerationException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
```

Comment:
Not done.
Can you please use php-cs-fixer, as suggested on our contributing file?